runtests.py

```diff
@@ -1077,7 +1100,8 @@ def find_module_source_file(self, source_file):
         return source_file
 
     def build_target_filename(self, module_name):
-        target = '%s.%s' % (module_name, self.language)
+        ext = self.language
+        target = '%s.%s' % (module_name, ext)
```

Comment:
I think it's clear enough the way it was.
```suggestion
        target = '%s.%s' % (module_name, self.language)
```